src/components/StylePopup/StylePopup.js

```diff
@@ -129,4 +122,13 @@ class StylePopup extends React.PureComponent {
   }
 }
 
-export default StylePopup;
\ No newline at end of file
+const mapStateToProps = (state,ownProps) => ({
+  colorPalette: selectors.getColorPalette(state, ownProps.annotationType),
+  annotationType: ownProps.annotationType
+});
+
+const mapDispatchToProps = {
+  setColorPalette: actions.setColorPalette,
+}
+
+export default connect(mapStateToProps, mapDispatchToProps)(StylePopup);
```

Comment:
Not sure if it would be better to connect `ColorPaletteHeader` with redux and pass `colorPalette` and `setColorPalette` to it directly instead of pass those two props through `StylePopup` (line 111)